main/tests/server/src/com/google/refine/tests/model/UrlFetchingTests.java

```diff
@@ -169,13 +169,13 @@ public void testUrlCaching() throws Exception {
 
         // Inspect rows
         String ref_val = (String)project.rows.get(0).getCellValue(1).toString();
-	if (ref_val.startsWith("HTTP error 403"))
+	if (ref_val.contains("HTTP error 403"))
             return;
```

Comment:
to skip the 500 error etc, I think:
if (ref_val.startsWith("HTTP error"))
would be better.

Reply:
sounds good (it does not actually work anyway but oh well)